Header header

bool battery_ok
bool motors_active
float32 loop_rate
bool is_ready
string robot_name
